test/integration/recursor_source/serverspec/recursor_source_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe 'A PowerDNS Resolver' do
+  it 'is running' do
+    expect(process('pdns_recursor')).to be_running
+  end
+
+  it 'can resolve example.com' do
+    expect(command('dig @localhost example.com').stdout).to \
```

Comment:
@therobot Really? That is interesting because it _should_ still work. That or add `8.8.8.8`